libsmartcols/src/filter-param.c

```diff
@@ -110,9 +111,14 @@ struct filter_node *filter_new_param(
 		return NULL;
 	}
 
+	n->percentage_holder = 0;
```

Comment:
I guess the new param is allocated by calloc(), so this is unnecessary, right?

Reply:
(Fixed locally).